types/scrivito/index.d.ts

```diff
@@ -49,8 +49,8 @@ interface OptimizeDefinition {
 
 export class Binary {
     private constructor();
-    static upload(source: Blob | File, options: { filename: string, contentType?: string }): FutureBinary;
-    static uplload(source: File, options?: { filename?: string, contentType?: string }): FutureBinary;
+    static upload(source: Blob | File, options: { filename: string; contentType?: string }): FutureBinary;
```

Comment:
Can you create a new interface for the upload options?

Reply:
done